src/api/upload.js

```diff
@@ -1,15 +1,17 @@
-import model from '../middleware';
+import upload from '../models/attendance_upload';
 
 module.exports = {
 
     attendance: (req, res, next) => {
-        model.upload.uploadAttendance(req, res).then((status) => {
+        upload.uploadAttendance(req, res).then((status) => {
             res.json({ message: status })
-        });
+        }).catch((err) => {
+            res.json(err);
+        })
```

Comment:
all 3 above files are inside api/ folder this includes routes and logic as well.

what is the actual purpose of api/ folder it not clear to me. please explain